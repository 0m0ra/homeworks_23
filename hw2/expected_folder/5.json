{
    "min age": 26,
    "max age": 55,
    "avg age": 40,
    "median age": 40,
    "< 2 days": 0,
    "< 1 week": 0,
    "< 1 month": 0,
    "< 6 months": 50.0,
    "> 6 months": 50.0
}