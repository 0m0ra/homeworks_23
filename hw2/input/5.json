{
    "user": {
        "region": "Saint-Petersburg",
        "email": "user@yandex.ru",
        "age": 36
    },
    "user2": {
        "region": "Sochi",
        "registered": "2022-10-30",
        "age": 18
    },
    "user3": {
        "region": "Moscow",
        "registered": "2019-10-30",
        "last_login": "2022-11-21",
        "email": "@mail.ru",
        "age": 18
    }
}