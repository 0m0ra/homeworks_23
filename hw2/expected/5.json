{
    "email": {
       "yandex.ru": 50.0,
       "mail.ru": 50.0
    },
    "registered": {
       "2019-10-30": 50.0,
       "2022-10-30": 50.0
    }
 }